challenges/objects-arrays.js

```diff
@@ -7,49 +7,75 @@
 */
 
 // tyrannosaurus, carnivorous, 7000kg, 12m, Late Cretaceous
+const tRex = {
+    name: 'tyrannosaurus',
+    diet: 'carnivorous',
+    weight: '7000kg',
+    length: '12m',
+    period: 'Late Cretaceous',
+    roar: "RAWERSRARARWERSARARARRRR!"
+}
 
 // stegosaurus, herbivorous, 2000kg, 9m, Late Jurassic
+const steg = {
+    name: 'stegosaurus',
+    diet: 'herbivorous',
+    weight: '2000kg',
+    length: '9m',
+    period: 'Late Jurassic'
+}
 
 // velociraptor, carnivorous, 15kg, 1.8m, Late Cretaceous
+const raptor = {
+    name: 'velociraptor',
+    diet: 'carnivorous',
+    weight: '15kg',
+    length: '1.8m',
+    period: 'Late Cretaceous'
+}
 
 // Using your dinosaur objects, log answers to these questions:
 
 // How much did tyrannosaurus weigh?
-console.log();
+console.log(tRex.weight);
 
 // What was the diet of a velociraptor?
-console.log();
+console.log(raptor.diet);
 
 // How long was a stegosaurus?
-console.log();
+console.log(steg.length);
 
 // What time period did tyrannosaurus live in?
-console.log();
+console.log(tRex.period);
 
 
 // Create a new roar method for the tyrannosaurus.  When called, return "RAWERSRARARWERSARARARRRR!" Log the result.
```

Comment:
Remember method is a function, so a roar method would look like: roar: function(){
return "roar";
} 
then you'r call it with tRex.roar()